src/VersionedContract.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+abstract contract VersionedContract {
+    function contractVersion() external returns (string memory) {
+        return "1.0.2";
```

Comment:
we're going to make this `1.0.0`, right?

Reply:
Yeah, probably a good idea.